test/bench/ok/heap-32.drun-run.ok

```diff
@@ -1,5 +1,5 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
-debug.print: (50_227, +91_377_860, 1_453_205_851)
-debug.print: (50_070, +102_586_000, 1_526_830_363)
+debug.print: (50_227, +91_377_860, 1_446_975_760)
```

Comment:
Is that test outputting instruction counts? Having to update expected test output that often is bad ergonomics, isn't it?

Reply:
Yes, you are right.